docs/guide/snaps-rpc-api.md

```diff
@@ -775,6 +775,70 @@ console.log(dogecoinPublicKey);
 
 ::::
 
+### `snap_getEntropy`
+
+::: warning Only Callable By
+
+- Snaps
+  :::
+
+#### Parameters
+
+- `Object`
+  - `version` - The literal number `1`. This is to allow for future versions of this method to be added.
+  - `salt` - An arbitrary string to be used as a salt for the entropy. This can be used to generate different entropy
```

Comment:
Should we add some more explicit note here about the determinism of the function? While it is described (nicely, I think) in the description, improper salt setting is a common cause of compromise and could warrant a notice for those who average library users who might not grok the implications?

Reply:
> improper salt setting and handling are common causes of compromise and could warrant a notice for those who might not grok the implications?

The entropy is Snap-specific. If two Snaps use the same salt, the generated entropy is still different. It shouldn't matter what salt is used.

The implementation is described here: https://github.com/MetaMask/SIPs/blob/60c212d1da607005bef989fd364565f21c66f111/SIPS/sip-6.md